auto3dseg/algorithm_templates/segresnet/scripts/segmenter.py

```diff
@@ -256,10 +269,20 @@ def get_resample_transforms(self, resample_label=True):
         return ts
 
     def get_normalize_transforms(self):
```

Comment:
Maybe we can consider adding some docstring for the functions in this script. I have difficulties to understand what the functions are for, such as `get_custom`. (Not a blocking issue)

Reply:
ok